src/explore-education-statistics-admin/src/index.tsx

```diff
@@ -12,12 +12,14 @@ process.env.APP_ROOT_ID = 'root';
 enableES5();
 configureAxios();
 
+getConfig().then(config => {
+  initApplicationInsights(config.AppInsightsKey);
```

Comment:
Do we still need any anonymous access to get this key prior to logging in?
I left `api/configuration/application-insights` in the controller just in case, but if not then we can remove it as well.

Reply:
Nope. I've removed it now